src/m_menu.c

```diff
@@ -3216,9 +3208,6 @@ setup_menu_t weap_settings2[] =
   {"Player View/Weapon Bobbing", S_CHOICE, m_null, M_X,
    M_Y + weap2_bobbing * M_SPC, {"cosmetic_bobbing"}, 0, M_UpdateCenteredWeaponItem, default_bobfactor_strings},
 
-  {"Enable Recoil Pitch", S_YESNO, m_null, M_X,
```

Comment:
I'm sure many people do like this.

Reply:
Reverted [here](https://github.com/fabiangreffrath/woof/compare/bf12f5c48a7e48ce04782eb900c97944507d8dd2..40c062a5e276cba94b540e2f7567e8cd359d3f6f).